src/Helpers/Utils.jsx

```diff
@@ -0,0 +1,5 @@
+
+
+export const removeSpaceFromStr = (str) => {
+    return str.replace(/\s/g,'')
+}
```

Comment:
Suggestions: you can use trim method to remove extra space from string.

Reply:
ok